espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/KReflectionUtils.kt

```diff
@@ -49,7 +49,7 @@ object KReflectionUtils {
                     if (classifier is KClass<*> && classifier.isSubclassOf(Class::class)) {
                         var className: String = providedParam.toString()
                         if (className.endsWith(".class")) {
-                            className = className.substring(0, className.length - ".class".length)
+                            className = className.take(className.length - ".class".length)
```

Comment:
why not takeLast?

Reply:
`takeLast` returns the elements in a string. I want the last 6 elements (the `.class` extension) removed.